packages/core/src/Toggle/Toggle.spec.tsx

```diff
@@ -32,4 +32,15 @@ describe('Toggle', () => {
     fireEvent.click(input)
     expect(cb).toHaveBeenCalled()
   })
+
+  test('add boxshadow when hover over the enabled states', () => {
+    const { getByLabelText, getByRole } = render(<Toggle label='Total price toggle' />)
+    const input = getByLabelText('Total price toggle')
+    fireEvent.mouseOver(input)
+    const container = getByRole('switch').parentNode
+    expect(container).toHaveStyleRule(
+      'box-shadow',
+      'rgba(0, 0, 0, 0.03) 0px -1px 0px 0px, rgba(0, 0, 0, 0.16) 0px 2px 8px 0px, rgba(0, 0, 0, 0.16) 0px 10px 8px -5px, rgba(0, 0, 0, 0.16) 0px 12px 32px -2px'
```

Comment:
Can you get this value from the theme rather than hardcoding it?

https://github.com/priceline/design-system/blob/main/packages/core/src/theme/theme.ts#L298